docs/configurations.md

```diff
@@ -26,6 +26,7 @@ This list contains configuration variables that can be used with all balena devi
 | BALENA_HOST_FIREWALL_MODE | string | false | off | Toggle firewall modes between on, off, and auto. | v11.9.1 |
 | BALENA_HOST_DISCOVERABILITY | boolean | false | true | Enable / Disable Avahi to run which will allow the device to respond to requests such as network scans. | v11.9.2 |
 | BALENA_HOST_SPLASH_IMAGE | integer | true | /boot/splash/balena-logo-default.png | Allows changing splash screen on boot to user defined file from userspace. | v12.3.0 |
+| BALENA_SUPERVISOR_DISABLE_HARDWARE_METRICS | boolean | false | false | Disables hardware metrics such as CPU temperature and memory usage from being reported to the backend and displayed in the dashboard. Useful for devices with bandwidth sensitivity. | v12.8.0 |
```

Comment:
```suggestion
| BALENA_SUPERVISOR_HARDWARE_METRICS | boolean | false | true | Periodically queries the device for hardware metrics such as CPU utilization, CPU temperatue, memory usage, and disk space. This info is then reported back to the cloud quite frequently (10 second intervals if there are changes) | v12.8.0 |
```

I am glad @pipex mentioned that this var name isn't like our existing ones. We usually have configurations for a feature which are default to true and can be explicitly set to true or false (see local mode, deltas, persistent logging...they don't say enable/disable they just say the name of the feature).

I'll mentioned we do have `BALENA_HOST_CONFIG_disable_splash` but that is a host config not Supervisor config so it might make sense for those but also, it might have been a mistake adding disable_splash.

Reply:
I went with `Toggle hardware metrics reporting to the cloud, which occurs every 10 seconds when there are changes. Metrics include CPU utilization, CPU temperature, memory usage, and disk space. Useful for devices with bandwith sensitivity.` as a description to combine your suggested change's intent and my original description's intent, since I'm not completely sold on your suggestion's wording. Lmk what you think